.github/workflows/ci.yml

```diff
@@ -20,14 +20,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # Nextflow versions
-        include:
-          # Test pipeline minimum Nextflow version
-          - NXF_VER: "21.10.3"
-            NXF_EDGE: ""
-          # Test latest edge release of Nextflow
-          - NXF_VER: ""
-            NXF_EDGE: "1"
+        NXF_VER:
+          - "21.10.3"
```

Comment:
I would update this to `22.10.1` at least, but stick to a stable version.
https://github.com/nextflow-io/nextflow/releases